docs/index.rst

```diff
@@ -99,26 +58,16 @@ package for plasma physics currently under development.
    development/release_guide
 
 .. toctree::
-   :maxdepth: 3
+   :maxdepth: 1
    :caption: All the Rest
-   :hidden:
 
    about/credits
    whatsnew/index
    about/stability
    PlasmaPy.org <https://www.plasmapy.org>
 
-
 .. The about PlasmaPy section has some important information that would
    be helpful to have more readily accessible from the main doc index
    page.
 
-*****
-Index
-*****
-
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
```

Comment:
Just curious - do you think there's no place for the module index besides the button in the top left?

Reply:
I don't think it needs to be anywhere else.  Where it is now will make it visible on all pages.  The only other location I can think to put the links would be in the top-right corner where the view page source link currently is.

I personally like it better in the sidebar because in makes more logical sense to me and would not be crowded by the navigation path (i.e. the `Docs » Formulary (plasmapy.formulary) » Distribution functions (plasmapy.formulary.distribution)`).

Reply:
Okay then :)